tools/cockpit.spec

```diff
@@ -516,6 +521,10 @@ getent group cockpit-wsinstance >/dev/null || groupadd -r cockpit-wsinstance
 getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance -d /nonexisting -s /sbin/nologin -c "User for cockpit-ws instances" cockpit-wsinstance
 
 %post ws
+%if 0%{?suse_version}
+%set_permissions %{_libexecdir}/cockpit-session
+%tmpfiles_create cockpit-tempfiles.conf
```

Comment:
This would  be an excellent thing to do on Fedora/RHEL as well. Do you mind moving it out of the `%if`? That would also not hurt to be a separate commit then.

Reply:
ok